File: 8KYU/Calculate BMI/solution.js
function bmi(weight, height) {
    var bmi = (weight / (height * height));
    if (bmi <= 18.5)
      return "Underweight";
    if (bmi <= 25)
      return "Normal";
    if (bmi <= 30)
      return "Overweight";
    
    return "Obese";
}

File: 8KYU/Calculate BMI/solution.py
def bmi(weight, height):
    bmi = weight / (height*height)
    if bmi <= 18.5:
        return "Underweight"
    elif bmi <= 25:
        return "Normal"
    elif bmi <= 30:
        return "Overweight"
    return "Obese"